content/en/docs/collector/build-connector.md

```diff
@@ -0,0 +1,544 @@
+
+    #### Parameters for the `createTracesToMetricsConnector`:
+
+    - `context.Context`: the reference to the Collector’s `context.Context` so
+      your trace receiver can properly manage its execution context.
+    - `connector.CreateSettings`: the reference to some of the Collector’s
+      settings under which your receiver is created.
+    - `component.Config`: the reference for the receiver config settings passed
+      by the Collector to the factory so it can properly read its settings from
+      the Collector config.
+    - `consumer.Metrics`: the reference to the next consumer type in the
+      pipeline, which is where received traces will go. This can be a processor,
+      exporter or another connector.
+
+4.  Write a `NewFactory` function that instantiates your custom factory for your
+    connector(component).
+
+             // NewFactory creates a factory for tailtracer connectors.
+             func NewFactory() connector.Factory {
+                 // OpenTelemetry connector factory to make a factory for connectors
+                 return connector.NewFactory(
+                 typeStr,
+                 createDefaultConfig,
+                 connector.WithTracesToMetrics(createTracesToMetricsConnector, component.StabilityLevelAlpha))
+             }
+
+factory.go file once finished:
+
+    package exampleconnector
+
+    import (
+        "context"
+
+        "go.opentelemetry.io/collector/component"
+        "go.opentelemetry.io/collector/connector"
+        "go.opentelemetry.io/collector/consumer"
+    )
+
+    const (
+        defaultVal = "request.n"
+        // this is the name used to refer to the connector in the config.yaml
+        typeStr = "example"
+    )
+
+
+    // NewFactory creates a factory for tailtracer connector.
+    func NewFactory() connector.Factory {
+        // OpenTelemetry connector factory to make a factory for connectors
+
+        return connector.NewFactory(
+        typeStr,
+        createDefaultConfig,
+        connector.WithTracesToMetrics(createTracesToMetricsConnector, component.StabilityLevelAlpha))
+    }
+
+
+    func createDefaultConfig() component.Config {
+        return &Config{
+            AttributeName: defaultVal,
+        }
+    }
+
+
+    // defines the consumer type of the connector
+    // we want to consume traces and export metrics, therefore, define nextConsumer as metrics, consumer is the next component in the pipeline
+    func createTracesToMetricsConnector(ctx context.Context, params connector.CreateSettings, cfg component.Config, nextConsumer consumer.Metrics) (connector.Traces, error) {
+      c, err := newConnector(params.Logger, cfg)
+      if err != nil {
+        return nil, err
+      }
+      c.metricsConsumer = nextConsumer
+      return c, nil
+    }
+
+## Implementing the Trace Connector
+
+Implement the methods from the interface component specific to the type of the
+component in the connector.go file. In this tutorial we will implement the
```

Comment:
nit: mention explicitly what interfaces you are implementing in this section by their name (e.g. `component.Component`)